core/ops_builtin_v8.rs

```diff
@@ -861,53 +854,49 @@ fn op_apply_source_map(
 /// `JsError::exception_message`. The callback is passed the error value and
 /// should return a string or `null`. If no callback is set or the callback
 /// returns `null`, the built-in default formatting will be used.
-#[op(v8)]
-fn op_set_format_exception_callback<'a>(
+#[op2(core)]
+pub fn op_set_format_exception_callback<'a>(
   scope: &mut v8::HandleScope<'a>,
-  cb: serde_v8::Value<'a>,
-) -> Result<Option<serde_v8::Value<'a>>, Error> {
-  let cb = to_v8_fn(scope, cb)?;
+  cb: v8::Local<'a, v8::Function>,
+) -> Option<v8::Local<'a, v8::Value>> {
+  let cb = v8::Global::new(scope, cb);
   let context_state_rc = JsRealm::state_from_scope(scope);
   let old = context_state_rc
     .borrow_mut()
     .js_format_exception_cb
     .replace(Rc::new(cb));
   let old = old.map(|v| v8::Local::new(scope, &*v));
-  Ok(old.map(|v| from_v8(scope, v.into()).unwrap()))
+  old.map(|func| func.into())
 }
 
 #[op(v8)]
 fn op_event_loop_has_more_work(scope: &mut v8::HandleScope) -> bool {
   JsRuntime::event_loop_pending_state_from_scope(scope).is_pending()
 }
 
-#[op(v8)]
-fn op_store_pending_promise_rejection<'a>(
-  scope: &mut v8::HandleScope<'a>,
-  promise: serde_v8::Value<'a>,
-  reason: serde_v8::Value<'a>,
+#[op2(core)]
+pub fn op_store_pending_promise_rejection(
+  scope: &mut v8::HandleScope,
+  promise: v8::Local<v8::Promise>,
```

Comment:
I added a place where we can make conversion to Globals automatic but didn't implement it -- I can circle back and implement this to reduce some more boilerplate here.

Reply:
I'm not sure I'm a fan of this solution - creating `Global`s is expensive, I think it shouldn't be hidden away.

Reply:
I think if that's the case, we should add a `#[global]` which draws the eye to the expense. I don't mind adding slightly more expensive boilerplate if we make it searchable and visible.

Reply:
+1 for attribute